packages/map-template/src/components/WebComponentWrappers/ListItemLocation/ListItemLocation.jsx

```diff
@@ -26,12 +27,17 @@ function ListItemLocation({ location, locationClicked, icon, isHovered }) {
             }
         }
         const unhoverHandler = () => {
-             // Check if the location is non-selectable before unhovering it
+            // Check if the location is non-selectable before unhovering it
             if (!location.properties.locationSettings?.selectable !== false) {
                 mapsIndoorsInstance.unhoverLocation(location);
             }
         }
 
+        // Add a disabled attribute to the non-selectable locations.
+        if (location.properties.locationSettings?.selectable === false) {
+            elementRef.current.setAttribute("disabled", "");
```

Comment:
Can we set a class name instead? The `disabled` attribute is usually used on form elements (inputs etc.). The `<mi-list-item-location>` element is not a form element, which may cause confusion.

Reply:
Good idea, replaced it with a class `non-selectable`.